Tests/Functional/TestMiscellaneous.swift

```diff
@@ -303,4 +303,18 @@ class MiscellaneousTestCase: XCTestCase {
             XCTAssertEqual(output, "Goodbye\n")
         }
     }
+
+    func testModuleWithTests() {
+        fixture(name: "SwiftTesting/SingleTarget", file: #file, line: #line) { prefix in
+            XCTAssertBuilds(prefix)
+            XCTAssertFileExists(prefix, ".build", "debug", "Foo.swiftmodule")
+            #if os(OSX)
+                XCTAssertDirectoryExists(prefix, ".build", "debug", "Package.xctest")
+            #else
+                //FIXME: test-Package not generated during swift-build on linux so this will fail.
+                //XCTAssertFileExists(prefix, ".build", "debug", "test-Package")
```

Comment:
`XCTAssertBuilds` calls `executeSwiftBuild` which runs `swift build`
`test-Package` executable is generated when "test" target is built